README.md

```diff
@@ -125,7 +126,10 @@ model = AutoModelForCausalLM.from_pretrained(
 
 ## Together API
 
-Evo will also be soon available via an API by [TogetherAI](https://www.together.ai/).
+Evo is available through TogetheAI with a [web UI](https://api.together.xyz/playground/language/togethercomputer/evo-1-131k-base), where you can interaface and generate DNA sequences with a chat-like interface.
```

Comment:
Fix typos: "TogetheAI" and "interaface"

Reply:
Also, we repeat interface twice, maybe just "where you can generate DNA sequences with a chat-like interface."

Reply:
ok, updated with new commit :) thanks!